cmd/cli/version.go

```diff
@@ -37,8 +39,9 @@ type remoteVersionGetter interface {
 type remoteVersion struct{}
 
 type remoteVersionInfo struct {
-	meshName string
-	version  *version.Info
+	meshName  string
+	namespace string
+	version   *version.Info
 }
 
 type versionInfo struct {
```

Comment:
I wonder if it makes more sense to make the `remoteVersionInfo` field a slice vs. maintaining a separate slice.